app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsFragment.kt

```diff
@@ -14,7 +14,7 @@ class NewsFragment @Inject constructor() : WolmoFragment<NewsPresenter>(), NewsV
 
     override fun init() {
         vNewsRecyclerView.layoutManager = LinearLayoutManager(activity)
-        presenter.getNews()
+        presenter.onViewAttached()
```

Comment:
you don't need to call this on the init() function. `onViewAttached()` is executed even before init